TradeItIosTicketSDK2/TradeItYahooTradePreviewViewController.swift

```diff
@@ -168,7 +161,7 @@ class TradeItYahooTradePreviewViewController: TradeItYahooViewController, UITabl
     }
 
     @IBAction func actionButtonTapped(_ sender: UIButton) {
-        if self.placeOrderResult != nil {
+        if self.dataSource?.isOrderPlaced == true {
```

Comment:
this seems a little silly. Is there a reason why we can't just `...isOrderPlaced {`?

Reply:
`Value of optional type 'Bool?' not unwrapped; did you mean to use '!' or '?'?`